templates/bundle/resources/usr/local/bin/liferay_entrypoint.sh

```diff
@@ -83,4 +85,16 @@ function start_monitor_liferay_lifecycle {
 	fi
 }
 
+function generate_thread_dump_now {
+	if [[ -s "${LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE}" ]]
```

Comment:
This way if the file was not created when the system started up, we would never be able to turn it on.

Reply:
It looks like my wording was not good on the ticket: If this file is set to something else than empty string, the container should start a background process at startup and check if this file exists every 60 seconds.

What I wanted to say is that if the LIFERAY_DOCKER_AUTO_THREAD_DUMP_SLEEP_FILE is set to something, then we should keep monitoring if the file is created (once every minute) and if it's created, we should make a thread dump.